config/constants.yml

```diff
@@ -511,6 +511,7 @@ funding_sources: { 'College Department': 'college',
                    'Student Funded Research': 'unfunded' }
 
 funding_statuses: { 'Pending Funding': 'pending_funding',
+                    'Undetermined': 'undetermined',
```

Comment:
This will cause conflicts with the new PermissibleValues branch. Ideally this branch should wait until #1119 is merged so that we can use the updated PermissibleValues formatting

Reply:
Agree.